CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## [1.0.0] - 2021-02-02
+
+### Added
+- Filtering by federated trust domains in `ListEntries` (#1967)
+- Logging when lookup of user by uid or group by gid fails in the `unix` WorkloadAttestor plugin (#2048)
+- Timeouts to calls to KeyManager plugins (#2044)
+- gRPC Health v1 services in Server and Agent exposed at `/grpc.health.v1.Health/Check` path (#2057, #2058, #2087)
```

Comment:
I'm wondering if we should mention that these are basic health services.

Reply:
By "basic", do you mean they are in an initial state and we plan to expand the implementations later?

Reply:
Yes. I think that it would make sense to somehow clarify that this implementation is the starting point for a more comprehensive solution that needs to be discussed (captured in #2047). Just to make sure that somebody reading the CHANGELOG is not confused thinking that this is the definite solution for the health system in SPIRE. Maybe something along the lines of "Initial implementation of gRPC Health v1 services in server and agent..." or "Basic implementation of..."?